doc/source/Tutorials/Scripting/CreatingConfigurationFiles3/index.md

```diff
@@ -0,0 +1,166 @@
+# Tutorial: Startup Configs, Part 3 # 
+
+In this final part of the startup config tutorial, we will demonstrate a startup config that adds a custom entry to the node menu.
+
+![A custom entry in the node menu](images/tutorialNodeMenuCustomEntry.png "A custom entry in the node menu")
+
+The entry will insert a Reference node and load a reference script into it. The sub-graph inside the Reference node consists of an OSLCode node that procedurally generates a texture of a Macbeth chart. This "custom" node should come in handy for lookdev shader development, and, as a bonus, provides a simple demonstration of procedural texture generation in OSL.
+
+Functionally, this node will be like a new node that came from a custom module. However, since the node is basically just an in-graph wrapper made of standard nodes, it will remain portable between Gaffer deployments, meaning that if it is saved inside a script, other users will not need to install additional modules in order to open that script.
```

Comment:
Sort of... they'll need to have access to the reference file, which is essentially a one-file module... I get the point you're making though. Maybe just clarify that the reference file isn't embedded into the new script, and will need to be available to users via a shared filesystem?